source/includes/rest/_subscribers.md

```diff
@@ -341,6 +351,14 @@ If you need to create or update a collection of subscribers at once, use our [ba
       <td><code>phone</code></td>
       <td>Optional. The subscriber's primary phone number.</td>
     </tr>
+    <tr>
+      <td><code>sms_number</code></td>
+      <td>Optional. String. The subscriber's mobile phone number in E.164 formatting. E.g. <code>"+16125551212"</code>. Only US-based numbers are supported at this time.</td>
+    </tr>
+    <tr>
+      <td><code>sms_consent</code></td>
+      <td>Optional. Boolean. Required if <code>sms_number</code> is not <code>null</code>. <code>true</code> if the person has granted consent to receive marketing and other communication via SMS; <code>false</code> otherwise. If you’re unsure whether or not you have gained legal SMS consent, check out our <a href="https://my.drip.com/docs/manual/sms/compliance-manage-sms-compliance">TCPA requirements article</a>.</td>
```

Comment:
I think we should remove the note about it being required. It's not anymore. We'll still create the SMS person for the number, but they'll be opted out.

Also, if they don't include `sms_consent` or set its value to `null`:
* Internally, we'll consider it `nil` by default but in some ways treat it the same as `false` (we'll still create the SMS person)
* Externally, it'll look like it was set to false (`json.sms_consent subscriber.sms_opted_in?`)

This is a little weird, and I'm not sure how to best describe that behavior in the contract. Maybe we just say that the default is `false` and `null` will be treated the same as `false`?